makefile

```diff
@@ -6,7 +6,7 @@ SOURCE_URL=https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
 
 
 # MAIN COMMANDS / GOALS ------------------------------------------------------------------------------------------------
-all: tmp/output/release/icd11foundation.owl
+all: tmp/output/release/icd11foundation.owl tmp/output/release/ordo-icd11.sssom.tsv
```

Comment:
## GitHub action?
- [ ] Make a determination @matentzn or @twhetzel 

Now there are two artefacts in [the release](https://github.com/monarch-initiative/icd11/releases/tag/v2024-03-30):
1. `icd11foundation.owl`: This is about the same as the source we download, but with unicode characters stripped.
2. `ordo-icd11.sssom.tsv`

Prior to this PR, the reason to not create a GitHub action is because the URL tied to `icd11foundation.owl` is to a static, past release, as opposed to a 'latest' release that gets updated regularly. We do want to change this if we can though; see:
- #8

In the case of `ordo-icd11.sssom.tsv`, though, this will be updatable every July, as that is when the nomenclature files are released. So we could theoretically have an action that runs and detects a new nomenclature file and then makes a release. We could set this to run every year in July/August. Or, since the exact release date is variable, and if we want to get the release out quickly after they do their release, we could set it this action to run like daily/weekly, and if it detects a new nomenclature release, it does its thing, else quits. So it would run often but only make a new release once a year.

Reply:
Going to check for updated nomenclature manually the next time it is released. Will make an issue to make a GH action.

Reply:
Created an issue for this:
- #13